gui-libs/amtk/amtk-5.6.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome2 meson
+
+DESCRIPTION="Actions, Menus and Toolbars Kit for GTK+ applications"
+HOMEPAGE="https://wiki.gnome.org/Projects/Amtk https://gitlab.gnome.org/World/amtk"
```

Comment:
I've added remote-id to all GNOME packages' `metadata.xml` files, so let's drop the gitlab URL from `HOMEPAGE`.

Reply:
Ah, never mind this. https://wiki.gnome.org/Projects/Amtk just contains a link to the gitlab repo. Let's remove the other URL instead.